.github/workflows/plurimath_js.yml

```diff
@@ -0,0 +1,22 @@
+name: Plurimath JS workflow
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  rake:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3'
+          bundler-cache: true
+      - uses: peter-evans/repository-dispatch@v3
+        with:
+          token: ${{ secrets.pat_token || github.token }}
```

Comment:
You have to use plurimath PAT token, I assume that PLURIMATH_CI_PAT_TOKEN should be defined

Reply:
Thanks @CAMOBAP, I’m looking into it now.

Reply:
Now I see the following error on the [workflow](https://github.com/plurimath/plurimath/actions/runs/10697689504/job/29655410441?pr=289),
> Error: Bad credentials

@CAMOBAP Is it possible that the PLURIMATH_CI_PAT_TOKEN is not defined?

Reply:
Yep, it's possible, @ronaldtse could you please confirm?

Reply:
Let me check.

Reply:
I've finally done it, sorry for the delay.

Reply:
It was defined but had to replace it with a new token.

Reply:
It works now -- the job passed! @suleman-uzair is this ready to go?

Reply:
> It works now -- the job passed! @suleman-uzair is this ready to go?

@ronaldtse I’ll get it ready soon and ping you here.